### PR TITLE
STAC-13966 add metric to flowlog relation

### PR DESCRIPTION
**Pull request by hruhek**
### Step 1: Link to Jira issue
https://stackstate.atlassian.net/browse/STAC-13966

### Step 2: Description of changes
Add metrics for bytes to Flow logs relation

### Step 3: Did you add / update tests for your changes in the right area?
- [x] Unit/Component test
- [ ] Integration test	

### Step 4: I'm confident that everything is properly tested:
I got a PO / QA Approval by:
- [ ] Name

### Step 5: Can we ship this feature to production?
- [ ] Yes, I'm proud of my work. Ship it! :ship:

### CODE REVIEW

File: aws_topology/stackstate_checks/aws_topology/flowlogs.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def __init__(self, namespace, family, laddr, raddr, start, end, traffic_type, in
     def add_traffic(self, start, end, traffic_type, incoming, byte_count, nwitf, reverse, log):
```
Comment (julianosk) [inline aws_topology/stackstate_checks/aws_topology/flowlogs.py:67]: Rename `nwitf` to `network interface`

**Review by julianosk — APPROVED** [review]
Just a tiny parameter renaming

File: aws_topology/stackstate_checks/aws_topology/aws_topology.py [hunk 2]
```diff
@@ -311,15 +318,20 @@ def finalize_account_topology(self):
     def event(self, event):
         self.agent.event(event)
 
+    def gauge(self, name, value, tags=None, hostname=None, device_name=None):
+        self.agent.log.info('gauge %s: %s %s', name, value, tags)
```
Comment (julianosk) [inline aws_topology/stackstate_checks/aws_topology/aws_topology.py:322]: Change this log to `debug`

Comment (hruhek) [reply aws_topology/stackstate_checks/aws_topology/aws_topology.py]: done

File: aws_topology/stackstate_checks/aws_topology/flowlogs.py (hunk 1 above)
Comment (hruhek) [reply aws_topology/stackstate_checks/aws_topology/flowlogs.py]: done

**Pull request merged**